chapter_12_organising_test_files.asciidoc

```diff
@@ -385,8 +378,8 @@ Brilliant--no need to sit around waiting for all the FTs
 when we're only interested in a single one.
 Although we need to remember to run all of them now and again, to check for regressions.
 Later in the book we'll set up a Continuous Integration (CI) server to run all the tests automatically,
-for example every time we push to master.
-// DAVID: "master" -> "our main branch"
+for example every time we push to the default branch (which is the `main` or
+formerly known as the `master` branch).
```

Comment:
I would keep a reference to `master`. As I checked (and found a URL pointing to the `master` tree in chapter 02) we refrained from naming the default branch so far. Also, the official git documentation still uses the original name for the default branch. So I think it's best to just call it default, and maybe mentioning the two most common names for it.

Reply:
it ends up quite wordy tho.  i quite like david's phrasing, which neatly leaves it ambiguous (if your default branch is called master, it's still your main branch, even if it's not literally named main).  i don't think people will be confused.